api/cotrip_app/models.py

```diff
@@ -6,3 +6,17 @@ class Topic(models.Model):
 
     def __str__(self):
         return self.title
+
+class Location(models.Model):
+    city = models.CharField(max_length=75)
+    state = models.CharField(max_length=30, null=True, blank=True)
+    country = models.CharField(max_length=50)
+    lat = models.DecimalField(max_digits=22,
+    decimal_places=16)
+    lon = models.DecimalField(max_digits=22,
```

Comment:
Can you use the full name for lat and long?

Reply:
Changed.